lib/matplotlib/patches.py

```diff
@@ -3520,18 +3524,19 @@ def __init__(self, head_length=.4, head_width=.2):
     _style_list["-|>"] = CurveFilledB
 
     class CurveFilledAB(_Curve):
-        """
-        An arrow with filled triangle heads both at the begin and the end
+        """An arrow with filled triangle heads both at the begin and the end
```

Comment:
This really should be one line. Maybe change to "at both ends" like `BracketAB` does below.